src/app/contact/contact-form/contact-form.component.scss

```diff
@@ -0,0 +1,129 @@
+.overlay {
+	position: fixed;
+	left: 0px;
+	top: 0px;
+	padding-top: 80px;
+	width: 100%;
+	height: 100%;
+	text-align: center;
+	z-index: 1000;
+	background-color: rgba(0, 0, 0, 0.5);
+
+	.overlay-content{
+		width: 620px;
+		height: 560px;
+		background-color: white;
+		margin: 0 auto;
+		border: solid gray 1px;
+		border-radius: 8px;
+		padding: 30px;
+		position: relative;
+
+		.form-close{
+			border: none;
+			width: 40px;
+			height: 40px;
+			font-size: 30px;
+			position: absolute;
+			right: 30px;
+			top: 15px;
+			background: white;
+			color: gray;
+			cursor: pointer;
+		}
+
+		.contact-form {
+			text-align: left;
+			
+		
+
+ 			input[type="text"],
+ 			input[type="email"],
```

Comment:
Actually I used radio inputs previously  and I didn't want to apply this rule to radio inputs. I forgot. Correcting it now :)